src/yetibot/core/logging.clj

```diff
@@ -1,17 +1,17 @@
 (ns yetibot.core.logging
   (:require
-    [schema.core :as s]
+    [clojure.spec.alpha :as s]
     [yetibot.core.config :refer [get-config]]
     [taoensso.timbre.appenders.3rd-party.rolling :refer [rolling-appender]]
     [taoensso.timbre.appenders.core :refer [println-appender]]
     [taoensso.timbre
      :as timbre
      :refer [trace debug info warn error fatal spy with-log-level]]))
 
-(def config-schema String)
+(s/def ::config string?)
 
 (defn log-level []
-  (let [v (get-config config-schema [:log :level])]
+  (let [v (get-config ::config [:log :level])]
```

Comment:
Also need to use a spec for the `get-config` call below on line 23: `(get-config String [:log :rolling :enabled])`. 

Otherwise it throws:

```
Execution error (ClassCastException) at yetibot.core.util.config/get-config (config.clj:55).
java.lang.Class cannot be cast to clojure.lang.IFn
```

Reply:
Good catch. I think when I tackled this I made the mistake of only looking for schema requires. Now looked for all calls to `get-config` and made sure it always gets a spec.